web/src/pages/stacklets/list.tsx

```diff
@@ -1,26 +1,26 @@
 import { Show, createResource } from 'solid-js';
-import { getProductClusters } from '../../api';
+import { getStacklets } from '../../api/index';
 import { DataTable } from '../../components/datatable';
 import { A } from '@solidjs/router';
 
-export const ProductClusters = () => {
-  const [productClusters, { refetch }] = createResource(getProductClusters);
+export const Stacklets = () => {
+  const [stacklets, { refetch }] = createResource(getStacklets);
   return (
     <>
       <button onClick={refetch}>Refresh</button>
-      <Show when={productClusters.loading}>Loading...</Show>
+      <Show when={stacklets.loading}>Loading...</Show>
       <DataTable
-        items={productClusters() || []}
+        items={stacklets() || []}
         columns={[
           {
             label: 'Product',
-            get: (cluster) => cluster.product,
-            sortBy: (cluster) => cluster.product,
+            get: (x) => x.product,
```

Comment:
I feel like `x` is kinda non-descriptive. Why don't we use `stacklet` or `stack`?

Reply:
Mostly, just that it's a lot of extra typing and visual noise when we already know from context (the `items` parameter) that we're working with a list of stacklets.

I'd consider it similar to `stacklets.map((x) => x.metadata.name)` vs `stacklets.map((stacklet) => stacklet.metadata.name)`.

I'm specifically not using `stack`, since we've talked before about using that to describe a whole use-case deployment (think: roughly the same scope as our demos).

Reply:
Yeah I agree that `stack` is not the right name in this context at all! Let's stick with `x` for now as the code can get a little cluttered with longer names (as you mentioned). If we decide we want something other than `x`, this will be a simple find and replace.